### DISCUSSION TITLE
Categorize/Organize Docs

### DISCUSSION THREAD

**Discussion by edge-rps** [remix-run/remix#3909] [Proposals]
I'm not sure if I'm the only one here that feels like the documentation site for remix is a little bit disorganized and takes a lot of reading to get started. Don't get me wrong, I'm very grateful for the provided docs. My proposition is an update to the docs page structure to streamline the experience for newcomers, I wanted to bounce the idea off my fellow remixers before making a PR.

### **My proposed doc structure**: 
_this not perfect by any means but just something to compare with current docs_
```
- Getting Started
     -   Installation (info about cli here and templates)
     -   Configuration (info about remix config file)
- Routing (page focused on everything to do with how routes work)
- Database (A vast exploration of diff providers fauna, sql, aws, ...etc and how to integrate with remix)
```

**Comment by herecydev** [comment]
Totally agree, some of the guides are intuitively presented, like styling or performance, but some others are really buried. An example would be sessions/cookies which could easily be a standalone guide, however it currently lives within the "Remix packages" API section. Another oddity is the guides vs pages? What's the difference? 

I do think the actual content within each page is great though, packed with lots of information.
